examples/python3/http_func.py

```diff
@@ -3,5 +3,5 @@
 import requests
 
 def handle(ctx, payload):
-    resp = requests.get("http://example.com")
+    resp = requests.get("http://www.example.com")
```

Comment:
Is this really necessary? It looks like `http://example.com` and `http://www.example.com` serve the same page.

Reply:
I just try again, it same, but on that day, I cannot access the url without www